gradle/ant-compat/folder-layout.gradle

```diff
@@ -25,11 +25,15 @@ allprojects {
       test.resources.srcDirs = ['src/test-files']
     }
 
-    task copyTestResources(type: Copy) {
+    task copyTestResources(type: Sync) {
       from('src/test') {
         exclude '**/*.java'
       }
       into sourceSets.test.java.outputDir
+      preserve {
+        include '**/*.class'
+        include 'META-INF/services/*'
```

Comment:
`Sync` deletes everything in the target which isn't in the source, and the target is a normal compiler output root, which contains class files etc. I've tried to enumerate things that shouldn't be deleted here, but I'm not terribly sure this list is exhaustive. I haven't seen issues with this settings so far, but I'm only working on Hunspell subset. What's worse, the issues I've seen before adding `preserve` appeared not immediately and somewhat randomly and weren't easy to decipher. So while the issue is there and should be addressed, I'm not sure this is the full fix.